rails-commenting.rb

```diff
@@ -5,27 +5,27 @@
 
 # app/controller/blog_posts_controller.rb
 
-# 1)
+# 1) The controller is the middle man that holds the logic which updates or searches for the information in the model to present to the view
 class BlogPostsController < ApplicationController
   def index
-    # 2)
+    # 2) gets all of the blogs posts from the model and send it to the view
     @posts = BlogPost.all
   end
 
   def show
-    # 3)
+    # 3) finds an individual blog post by passing the id to the model and sends it to the view
     @post = BlogPost.find(params[:id])
   end
 
-  # 4)
+  # 4) rails convention for the route needed to display a form but doesn't do anything with the model/view. a midway point between C and R in CRUD.
```

Comment:
Totally. I think I would also elaborate as to why the method is empty and why its OK for it to be.